srcpkgs/sequoia-sq/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'sequoia-sq'
+pkgname=sequoia-sq
+version=0.27.0
+revision=1
+wrksrc="${pkgname}-v${version}"
+build_wrksrc="sq"
+build_style=cargo
+_deps="nettle-devel openssl-devel"
+hostmakedepends="pkg-config llvm clang ${_deps}"
+makedepends="${_deps}"
+short_desc="Command-line frontend for Sequoia, a new OpenPGP implementation"
+maintainer="Jan Christian Grünhage <jan.christian@gruenhage.xyz>"
+license="GPL-2.0-or-later"
+homepage="https://sequoia-pgp.org/projects/#sq"
+distfiles="https://gitlab.com/sequoia-pgp/sequoia/-/archive/sq/v${version}/sequoia-sq-v${version}.tar.gz"
+checksum=e943528be6af2b14c5b9d3a011335233d985ee69f43700d29d99908359f8c345
+conflicts="squirrel"
+
+case "$XBPS_TARGET_MACHINE" in
+	armv*l) nocross="Requires C libs included in build.rs, which is currently broken in xbps-src. These failures only manifest on a hf archs right now";;
```

Comment:
We talked on IRC about fixing this by updating the bindgen crate, right?

Reply:
Yeah. I tried it with `EXTRA_CLANG_ARGS_<TARGET>` after updating bindgen to 0.59, which I had hoped would help, but without success. The problems persisted

Reply:
I tested a native build on armv7l and it seems to work, so `nocross` seems like the right choice here.